src/containers/viewRequests/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import PFRequestsTable from '../../components/PFRequestsTable';
+
+const ViewRequests = () => {
+  return (
+    <>
+      <Helmet>
+        <title>View Family Details</title>
+        <meta
+          name="description"
+          content="View all pending participating family requests and current registered families"
+        />
+      </Helmet>
+      <PFRequestsTable />
```

Comment:
Data should be passed to the table as props from here to keep a separation of concerns, with components being presentational and containers handling complex logic.